docs/timelion/getting-started/timelion-conditional.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[timelion-conditional]]
+=== Using conditional logic and tracking trends
+
+In this section you will learn how to modify a series with conditional logic and trend your data with a moving average. This is helpful to easily distinguish outliers and detect trends over time.
+
+For the purposes of this tutorial, you will continue to use https://www.elastic.co/downloads/beats/metricbeat[Metricbeat data] to add another visualization that monitors memory consumption. To start, use the following expression to chart the maximum value of  `system.memory.actual.used.bytes`.
+
+[source,text]
+----------------------------------
+.es(index=metricbeat-*, timefield='@timestamp', metric='max:system.memory.actual.used.bytes')
+----------------------------------
+
+image::images/timelion-conditional01.png[]
+{nbsp}
+
+Let’s create two thresholds to keep an eye on the amount of memory your machine is using. For the purposes of this tutorial, your warning threshold will be 12.5GB and your severe threshold will be 15GB. When the maximum amount of memory being utilized exceeds either of these thresholds, the series will be re-styled accordingly.
```

Comment:
What does it mean by "the series will be re-styled"? Do we mean that they will have different colors?